splink/linker.py

```diff
@@ -2285,6 +2286,81 @@ def precision_recall_chart_from_labels_table(
         recs = df_truth_space.as_record_dict()
         return precision_recall_chart(recs)
 
+    def accuracy_chart_from_labels_table(
+        self,
+        labels_splinkdataframe_or_table_name,
+        threshold_actual=0.5,
+        match_weight_round_to_nearest: float = None,
+        add_metrics: list = [],
```

Comment:
There's a slight bug here if you input a string instead of a list:
```
linker.accuracy_chart_from_labels_column("group", add_metrics="phi")
```

You can also use a random string - `linker.accuracy_chart_from_labels_column("group", add_metrics="phaaai")` - to generate additional lines equal that scale with the length of the string.

![Screenshot 2023-08-03 at 20 50 01](https://github.com/moj-analytical-services/splink/assets/45356472/79838550-70a6-4e53-9699-5fc65d37ad3c)

Reply:
A quick check should fix it `isinstance(add_metrics, list)`

Reply:
Thanks for making me accountable, I was willing to ignore that 😅 
https://github.com/moj-analytical-services/splink/pull/1478/commits/84b15510218692c636839af526be33f78681f4c2  includes that check as well as doing a bit of tidying up. 

For example:
```
linker.accuracy_chart_from_labels_column("group", add_metrics=["precision", "recall", "NPV", "foo"])
```
will ignore "precision", "recall" (both already included as a minimum) and "foo" (not a valid option), and correct "NPV" to "npv".